src/showAPopUpScreen.ts

```diff
@@ -0,0 +1,16 @@
+export const showAPopUpScreen = (screenToDisplay: HTMLDivElement, startingPosition: string) => {
+    // display the help screen
+     screenToDisplay.style.display = startingPosition;
+    // function which gets called when the exit button is clicked
+    const exitButton = screenToDisplay.querySelector('.exitButton');
+    if (exitButton) {
+         const exitPopUpFunction = () => {
+             //hide the screen
+             screenToDisplay.style.display = 'none';
+             // remove the event listener
+             exitButton?.removeEventListener('click', exitPopUpFunction);
+     };
+     // add event listener to the exit button;
+     exitButton?.addEventListener('click', exitPopUpFunction);
+     }
```

Comment:
Please, format it. I got mindfuck because I thought that `const exitPopUpScreen` is declared only inside of `if` and adding event listener is outside of `if`- and it was mindfucking, because of scope of `const exitPopUpScreen` constant.

Reply:
you can use autoformatter :)